content/md/en/docs/tutorials/build-a-blockchain/simulate-network.md

```diff
@@ -169,7 +169,7 @@ To add a node to the running blockchain:
    --chain local \
    --bob \
    --port 30334 \
-   --rpc-port 9946 \
+   --rpc-port 9946 && \
    --rpc-port 9934 \
```

Comment:
```suggestion
   --rpc-port 9946 \
```

Reply:
Thank you for the input @ghe0. I knew the nodes could start with only rpc port 9945, but I thought the problem was deeper than that, as far configuring the CLI itself and the runtime. Consensus is we move on without rpc ports 9933 && 9934.

Reply:
Yes I think this is fine.

However we need to ensure we update the rest of the docs on this page to match. Please check to make sure we don't mention these removed ports anywhere else, and also mention that *both* RPC and WS is exposed on `--rpc-port`.